tests/Unit/Facade.Tests/SernosNoteServiceSpecs/ContextBase.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Linn.Products.Facade.Tests.SernosNoteServiceSpecs
+{
+    using Linn.Common.Persistence;
+    using Linn.Products.Domain.Linnapps;
+    using Linn.Products.Facade.Services;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class ContextBase
+    {
+        protected SernosNoteService Sut { get; private set; }
+
+        protected IRepository<SernosNote, int> SernosNoteRepository { get; private set; }
+
+        protected ITransactionManager TransactionManager { get; private set; }
+
+        [SetUp]
+        public void SetUpContext()
+        {
+            this.SernosNoteRepository = Substitute.For<IRepository<SernosNote, int>>();
+            this.TransactionManager = Substitute.For<ITransactionManager>();
+            this.Sut = new SernosNoteService(this.SernosNoteRepository, this.TransactionManager);
+        }
```

Comment:
test?

Reply:
Just realised these comments were on the context base which just sets up the sernos note tests (why it doesn't have tests itself)

Reply:
Ah Okay! 👍